interop-testing/src/main/java/io/grpc/testing/integration/AbstractTransportTest.java

```diff
@@ -628,6 +632,38 @@ public void gracefulShutdown() throws Exception {
     verifyNoMoreInteractions(responseObserver);
   }
 
+  /** Sends a large unary rpc with compute engine credentials. */
+  public void computeEngineCreds(String serviceAccount, String oauthScope) throws Exception {
+    ComputeEngineCredentials credentials = new ComputeEngineCredentials();
+    TestServiceGrpc.TestServiceBlockingStub stub = TestServiceGrpc.newBlockingStub(channel)
+        .configureNewStub()
+        .addInterceptor(new ClientAuthInterceptor(credentials, testServiceExecutor))
+        .build();
+    final SimpleRequest request = SimpleRequest.newBuilder()
+        .setFillUsername(true)
+        .setFillOauthScope(true)
+        .setResponseSize(314159)
+        .setResponseType(PayloadType.COMPRESSABLE)
+        .setPayload(Payload.newBuilder()
+            .setBody(ByteString.copyFrom(new byte[271828])))
+        .build();
+
+    final SimpleResponse response = stub.unaryCall(request);
+    assertEquals(serviceAccount, response.getUsername());
```

Comment:
However, I actually question how appropriate it is to do an assertequals on the full response. If the service account is in extra oauth scopes then the test will fail. It is probably best to not do the assertEquals.

I think I'm going to send an update to that test to remove the requirement for the assert on payload, since that isn't actually useful for the test.